configs/katello/4.7.yaml

```diff
@@ -0,0 +1,110 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.7:
+    :redmine_version_id: 1613
```

Comment:
I did notice that the due date is in the future for this: https://projects.theforeman.org/versions/1613

Reply:
That was leftover from when we made the release, noticed the description said placeholder for katello 4.7, changed the description to just Katello 4.7 and set the target date to be Nov 8th when Foreman branches.